flask_app/backend_jobs/ingestion_pipeline/providers/image_provider_flickr.py

```diff
@@ -80,6 +82,8 @@ def get_url_for_min_resolution(self,  min_height, min_width, image):
         # in flickr the resolution is represented in the url.
         # See details at https://www.flickr.com/services/api/misc.urls.html
         url = image['url']
+        if url == None:
```

Comment:
change to `if url is None`: https://google.github.io/styleguide/pyguide.html#2144-decision

Reply:
fixed